server/views/pages/appointments/video-link-booking/court/check-booking.njk

```diff
@@ -143,7 +143,8 @@
                             }
                         }
                     ] if session.bookACourtHearingJourney.preLocationCode else [{ value: { text: "No pre-court hearing" } }],
-                    classes: "govuk-summary-list govuk-summary-list--no-keys" if not session.bookACourtHearingJourney.preLocationCode
+                    classes: "govuk-summary-list govuk-summary-list--no-keys" if not session.bookACourtHearingJourney.preLocationCode,
+                    attributes: { 'data-qa': 'pre-scheduling-information' }
```

Comment:
These attributes look like new additions, what do they actually do?  Just for my benefit really.

Reply:
The give the integration test code an attribute to search for and verify the contents of the page - in this case we needed to check the content for the main, pre and post hearings separately.